type-o-matic/Program.cs

```diff
@@ -66,10 +75,10 @@ public static int Main (string [] args)
 			aggregatedTypes.Aggregate ();
 
 			var writer = new CodeWriter (options.OutputWriter);
-			if (options.Namespaces.Count > 0) {
+			if (options.ToGenerate.Equals ("swift")) {
```

Comment:
Better to use this version of the method: https://docs.microsoft.com/en-us/dotnet/api/system.string.equals?view=netframework-4.8#System_String_Equals_System_String_System_String_System_StringComparison_

Reply:
Fixed!